### PR TITLE
Docs Content Updates

### PR DESCRIPTION
**Pull request by timetraveler328**
- Use consistent sentence structure, articles, and punctuation.
- Use consistent format for endpoint descriptions (They should now all start with "Get").
- Add description content and keywords for better SEO
- Updated examples to use dates from this year.
- Got rid of the warnings that this call may crash your browser.
- Add internal and external links with SEO-friendly text and alt descriptions.
- General grammar/wording fixes and cleanup

### CODE REVIEW

File: _spec_/parameters.yml [hunk 1]
```diff
@@ -80,28 +81,30 @@ AggregateSort:
...
   schema:
     type: string
-  example: "2020-02-01"
+  example: "2020-10-14"
 ReverseOrder:
   name: reverse
   in: query
   description: |
-    Reverse the order of the results. This is useful in combination with `timestamp` param.
+    Reverse the order of the results. This is useful in combination with `timestamp` parameter.
```
Comment (qrpike) [inline _spec_/parameters.yml:99]:
```suggestion
    Reverse the order of the results.
```

**Review by qrpike — APPROVED** [review]
Ayyyyyyy very nice

File: _spec_/paths/crypto/aggregates.yml [hunk 2]
```diff
@@ -13,7 +16,7 @@ parameters:
   - $ref: '../../parameters.yml#/AggregateSort'
 responses:
   200:
-    description: Previous day OHLC for ticker
+    description: Cryptocurrency Aggregates.
```
Comment (jbonzo) [inline _spec_/paths/crypto/aggregates.yml:19]: Curious on your thoughts but I haven't seen us refer to this asset type as anything other than "crypto" on our website.

**Review by jbonzo — APPROVED** [review]

File: _spec_/paths/crypto/aggregates.yml (hunk 2 above)
Comment (timetraveler328) [reply _spec_/paths/crypto/aggregates.yml:19]: Yeah that's a good question, I chose "Cryptocurrency" cause that's the terminology we use on the marketing site page (https://polygon.io/crypto), but we currently don't really track anything having to do with crypto for SEO purposes and it rarely shows up in our organic search, so I'm not sure which term is more popular/better for searching.  Definitely open to changing it

Comment (jbonzo) [reply _spec_/paths/crypto/aggregates.yml:19]: I have no alternatives, I just noticed it felt a little inconsistent. We can trial it and change it as needed in the future

**Pull request merged**